ios/FluentUI/Core/Colors2.swift

```diff
@@ -12,166 +12,110 @@ import UIKit
         brandColors[.brandStroke1] = brandStroke1
     }
-    if let brandStroke1Pressed = provider.brandStroke1Pressed(for: theme)?.dynamicColor {
+    if let brandStroke1Pressed = provider.brandStroke1Pressed(for: themeable)?.dynamicColor {
         brandColors[.brandStroke1Pressed] = brandStroke1Pressed
     }
-    if let brandStroke1Selected = provider.brandStroke1Selected(for: theme)?.dynamicColor {
+    if let brandStroke1Selected = provider.brandStroke1Selected(for: themeable)?.dynamicColor {
         brandColors[.brandStroke1Selected] = brandStroke1Selected
     }
     return brandColors
 }
 
 // MARK: Colors
 
-private enum BrandColorsForOverriding: CaseIterable {
-    case brandBackground1
-    case brandBackground1Pressed
-    case brandBackground1Selected
-    case brandBackground2
-    case brandBackground2Pressed
-    case brandBackground2Selected
-    case brandBackground3
-    case brandBackgroundTint
-    case brandBackgroundDisabled
-    case brandForeground1
-    case brandForeground1Pressed
-    case brandForeground1Selected
-    case brandForegroundTint
-    case brandForegroundDisabled1
-    case brandForegroundDisabled2
-    case brandStroke1
-    case brandStroke1Pressed
-    case brandStroke1Selected
-
-    var equivalentAliasToken: AliasTokens.ColorsTokens {
-        switch self {
-        case .brandBackground1:
-            return .brandBackground1
-        case .brandBackground1Pressed:
-            return .brandBackground1Pressed
-        case .brandBackground1Selected:
-            return .brandBackground1Selected
-        case .brandBackground2:
-            return .brandBackground2
-        case .brandBackground2Pressed:
-            return .brandBackground2Pressed
-        case .brandBackground2Selected:
-            return .brandBackground2Selected
-        case .brandBackground3:
-            return .brandBackground3
-        case .brandBackgroundTint:
-            return .brandBackgroundTint
-        case .brandBackgroundDisabled:
-            return .brandBackgroundDisabled
-        case .brandForeground1:
-            return .brandForeground1
-        case .brandForeground1Pressed:
-            return .brandForeground1Pressed
-        case .brandForeground1Selected:
-            return .brandForeground1Selected
-        case .brandForegroundTint:
-            return .brandForegroundTint
-        case .brandForegroundDisabled1:
-            return .brandForegroundDisabled1
-        case .brandForegroundDisabled2:
-            return .brandForegroundDisabled2
-        case .brandStroke1:
-            return .brandStroke1
-        case .brandStroke1Pressed:
-            return .brandStroke1Pressed
-        case .brandStroke1Selected:
-            return .brandStroke1Selected
-        }
-    }
-}
-
 public extension FluentTheme {
-    /// Associates a `ColorProvider2` with a given `FluentTheme` instance.
+    /// Associates a `ColorProvider2` with a given `FluentThemeable` instance.
```

Comment:
Should this still be an extension of `FluentTheme`? Seems a bit odd to pass something with a theme into a theme to get colors from a theme.

Reply:
It's now an extension on `UIView`, which matches with normal `fluentTheme` setting and also clarifies the calling code.